sprint0/sprint0/Dungeon/DungeonSprites.cs

```diff
@@ -0,0 +1,120 @@
+                dungeonFactory.MakeSprite("right wall", new Vector2(600, 150)),
+            };
+            return wallSprites;
+        }
+
+        private List<ISprite> LoadDoorSprites()
+        {
+            List<ISprite> doorSprites = new List<ISprite>
+            {
+
+                dungeonFactory.MakeSprite("up open door", new Vector2(640, 150)),
+                dungeonFactory.MakeSprite("left open door", new Vector2(640, 150)),
+                dungeonFactory.MakeSprite("right open door", new Vector2(640, 150)),
+                dungeonFactory.MakeSprite("right locked door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("left locked door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("up locked door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("down shut door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("right shut door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("left shut door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("up shut door", new Vector2(680, 150)),
+                dungeonFactory.MakeSprite("down shut door", new Vector2(680, 150)),
+            };
+            return doorSprites;
+        }
+
+        private List<ISprite> LoadOpeningSprites()
+        {
+            List<ISprite> openingSprites = new List<ISprite>
+            {
+                dungeonFactory.MakeSprite("left bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("right bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("up bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("down bombed opening", new Vector2(720, 150)),
+            };
+            return openingSprites;
+        }
+
+        private List<ISprite> LoadDungeonBorderSprites()
+        {
+            List<ISprite> openingSprites = new List<ISprite>
+            {
+                dungeonFactory.MakeSprite("left bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("right bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("up bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("down bombed opening", new Vector2(720, 150)),
+            };
+            return openingSprites;
+        }
+
+        private List<ISprite> LoadDungeonFloorSprites()
+        {
+            List<ISprite> openingSprites = new List<ISprite>
+            {
+                dungeonFactory.MakeSprite("left bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("right bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("up bombed opening", new Vector2(720, 150)),
+                dungeonFactory.MakeSprite("down bombed opening", new Vector2(720, 150)),
+            };
+            return openingSprites;
```

Comment:
just realized that I forgot to take these out. ignore the dugeonborder and dungeonfloor methods